packages/gamut-labs/src/ListSection/index.tsx

```diff
@@ -34,11 +40,13 @@ export const ListSection: React.FC<ListSectionProps> = ({
   headerButton,
   headerSecondaryButton,
   initialDisplayAmount = 3,
+  onShowAllOrLessClick,
   children,
 }) => {
   const [showAll, setShowAll] = useState(false);
 
   const handleShowAll = () => {
+    if (onShowAllOrLessClick) onShowAllOrLessClick(!showAll);
```

Comment:
Nit: I'm irked this isn't in the latest batch of typescript-eslint rules we enabled... irked!

```suggestion
    onShowAllOrLessClick?.(!showAll);
```

Reply:
Ha! I had actually tried to do this earlier, and had forgotten that for optional chaining on a function call you still require the period, so I was running into `onShowAllOrLessClick?(!showAll);` erroring out thinking it was a ternary. I was very confused. Let this be a lesson to me to investigate confusion moments more deeply. Thanks for catching this!